api/classes/utils/wp-functions.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Some useful functions taken directly from the WordPress codebase.
```

Comment:
Since we're already including `wp-db.php`, it's probably worth it to also include `wp-includes/functions.php` instead of copy+pasting the functions.

Reply:
I was initially wary of including `functions.php` since that file requires `options.php`, but it seems to be working well. I had to rework the `ABSPATH` handling a bit to get this change working correctly on Atomic sites. https://github.com/Automattic/newspack-popups/pull/208/commits/7d987634230e192af15e298564b97ec6e6b74d3c